src/Behat/Context/FileContext.php

```diff
@@ -46,4 +46,45 @@ public function createFileWithName($filename, $directory = NULL) {
     $this->files[] = $this->getCore()->createFileWithName($path, $directory);
   }
 
-}
\ No newline at end of file
+  /**
+   * Get path for file in drupal 8.
+   *
+   * @param string $filename
+   *   The name of the file to get.
+   * @param string directory
+   *   A string containing the files scheme, usually "public://".
+   *
+   * @throws Exception
+   *   Exception file not found.
+   *
+   * @throws Exception
+   *   Exception destination not found.
+   *
+   * @Given I visit file with name :filename
+   * @Given I visit file with name :filename in the :directory directory
+   */
+  public function visitFileWithName($filename, $directory = NULL) {
+    $public = 'public://';
+    $private = 'private://';
+
+    if (empty($directory) || strpos($directory, $public) !== FALSE) {
+      $realpath = \Drupal::service('file_system')->realpath($directory);
```

Comment:
@keopx I forgot it, but this needs to be compatible with drupal 7. It's mentioned by @jorgetutor  here https://github.com/Metadrop/behat-contexts/pull/59#issuecomment-537842292. 

All steps are compatible with both d7 and d8. You can find an example about how to do it in method `createFileWithName`.

Reply:
Now is compatible with both version. Pending to implement method in Drupal 7.